glusterd2/volume/store-utils.go

```diff
@@ -120,6 +119,28 @@ func GetVolumes() ([]*Volinfo, error) {
 	return volumes, nil
 }
 
+// AreReplicateVolumesRunning retrieves the volinfo objects from GetVolumes() function
+// and checks if all replicate, Disperse volumes are stopped before
+// stopping the self heal daemon.
+func AreReplicateVolumesRunning() bool {
+	volumes, e := GetVolumes()
+	if e != nil {
+		return false
+	}
+	for _, v := range volumes {
+		if v.Type != Replicate && v.Type != Disperse {
```

Comment:
include two more types `DistReplicate` and `DistDisperse`)

Reply:
Resolved.